events/events-team/operations/sig-meeting-room-proctor.md

```diff
@@ -0,0 +1,18 @@
+# Contributor Summit SIG Meeting Moderator Instructions
+
+Thanks for volunteering to moderate a general session in the main presentation room! 
+Your contribution will make sure everyone can share knowledge and collaborate.
+SIGs can reserve a room for a SIG meeting and discussion at the Contributor Summit.
+
+To be a successful moderator, you’ll complete the following tasks:
+
+* Arrive at least 5 minutes before the session is scheduled to start
+* Ask the SIG if they would requested any AV and verify AV is working like microphone(s) or projector. Help troubleshoot any problems with AV, facilities, or event staff.
+* Timekeeping is valuable to make sure the day's schedule goes as planned. Make sure the session both begins and ends on time. Provide a time check to the session leader at 5/2/1 minute warnings (before the session is scheduled to end). At the latest, ensure you end 3 minutes before the start of the next session block.
+* Make sure the room has appropriate supplies e.g. pens, paper, etc
+* As a SIG meeting may be an official SIG meeting, SIGs should facilitate note taking if applicable
+* When it comes to crowd participation, don't let one person monopolize the conversation – pay attention to the discussion and politely encourage and highlight other people who have may have things to add
```

Comment:
I know this was in our older instructions, but I'm wondering if it's a realistic one to give.  The proctor isn't running the meeting, so they have no control over who speaks or not.

Reply:
I removed 

> * When it comes to crowd participation, don't let one person monopolize the conversation – pay attention to the discussion and politely encourage and highlight other people who have may have things to add